src/scenes/AddEdit/recurrence-selector.jsx

```diff
@@ -243,6 +258,8 @@ export default class RecurrenceSelector extends React.Component {
               {month_options}
               {week_options}
               <EndOptions option={this.state.end_option} onChange={this.endOptionsChanged}/>
+              <PlainEnglishRecurrence recurrence={this.state.recurrence} start={this.state.start}/>
+              <br/>
```

Comment:
Page breaks can be done with CSS (so you can control them, e.g. for different screen sizes). If you want an element to take up the entire width of its parent and push other elements to the next line, use `display: block;`.